README.md

```diff
@@ -134,9 +147,71 @@ This will fetch the latest version of the container, redeploy if a newer version
 
 #### Stopping the Container
 
-`docker-compose down` # From the source directory you started from
+From the source directory you started from run:
+
+```bash
+docker-compose down
+```
+
+---
+
+### Heimdall Server - Native
```

Comment:
This should probably include some indication that this is not the recommended way to run the application, something like `(Advanced)` next to the heading.